install.sh

```diff
@@ -92,7 +92,8 @@ https://hosts-file.net/ad_servers.txt
 https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext
 EOF
                 echo "[i] File created."
-                chmod u=rwx,g=rw,o=rw "hostssources.lst" # Allows the user to edit the file which is created by root.
+
+                #chmod u=rwx,g=rw,o=rw "${SCRIPT_DIR}/hostssources.lst" # Allows the user to edit the hostssources.lst file which is created by root, uncomment if needed.
```

Comment:
thanks for your changes. I removed this comment, because I don't want to recommend this security risk. Someone who wants to install adaway needs root permissions and if you have root permissions, you can edit this file.

Additionally I don't understand why you want to make this file executable...